bench-tps/src/bench_tps_client/rpc_client.rs

```diff
@@ -9,8 +10,18 @@ use {
 
 impl BenchTpsClient for RpcClient {
     fn send_transaction(&self, transaction: Transaction) -> Result<Signature> {
-        RpcClient::send_transaction(self, &transaction).map_err(|err| err.into())
+        RpcClient::send_transaction_with_config(
+            self,
+            &transaction,
+            RpcSendTransactionConfig {
+                skip_preflight: true,
```

Comment:
Even if `RpcClient` isn't the "default" type, I still think we should configure it to run as well as possible / in a consistent manner with other clients. Is your comment here in opposition to `skip_preflight == true` specifically ?

Reply:
ya we could just set the `preflight_commitment` and then leave the rest of `RpcSendTransactionConfig` to default, which would set `skip_preflight` to false

Reply:
yes, although `skip_preflight` will probably give higher tps, it will make errors less informative. If we have any plans to use RpcClient for testing, it might make sense but otherwise I would leave it with preflight checks

Reply:
> yes, although skip_preflight will probably give higher tps, it will make errors less informative

Makes sense. In that case, yeah, I'm onboard with leaving `skip_preflight == false`; we could also introduce a CLI flag to skip preflight in the future if we really want that case